utils.js

```diff
@@ -4266,14 +4479,39 @@ export function repoMetadataToGitHubApiUrl(repoMetadata) {
     const name = repoMetadata.name;
     let ghUrl = "https://api.github.com/repos";
     if (group && group !== "." && group !== "") {
-      ghUrl = `${ghUrl}/${group.replace("github.com/", "")}`;
+      ghUrl = `$ghUrl/${group.replace("github.com/", "")}`;
     }
     ghUrl = `${ghUrl}/${name}`;
     return ghUrl;
   }
   return undefined;
 }
 
+/**
+ * Method to run cli command `pixi install`
+ *
+ *
+ */
+export function generatePixiLockFile(path) {
+  const result = spawnSync("pixi", ["install"]);
+
+  if (result.status !== 0) {
+    // Handle errors
+    if (result.error && result.error.code === "ENOENT") {
+      console.error(
+        "Error: pixi command not found. Make sure pixi.js is installed globally.",
+      );
+    } else {
+      console.error(
+        `Error executing pixi install: ${result.error || result.stderr.toString()}`,
```

Comment:
Without utf-8 encoding, I think we will get an instance of Buffer back, so a simple `toString` may not work.